airbyte-webapp/src/components/base/Input/Input.tsx

```diff
@@ -102,7 +102,7 @@ const Input: React.FC<InputProps> = (props) => {
   }, [inputRef, defaultFocus]);
 
   return (
-    <InputContainer {...props} className={focused ? "input-container--focused" : undefined}>
+    <InputContainer className={focused ? "input-container--focused" : undefined}>
       <InputComponent
         {...props}
```

Comment:
@edmundito need your advice. Here we pass the props to `InputContainer` and `InputComponent`. Which causes firing `onChange` twice. Do we really need to rethrow props to `InputContainer`?